package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ghost",
-  "version": "2.1.1",
+  "version": "2.2.0",
```

Comment:
There shouldn't be any need to bump the version here. If it was bumped so that the migration worked you can force migrations without bumping package.json with `knex-migrator migrate --v 2.2 --force`

Reply:
Oh nice - thanks :+1: